### ISSUE TITLE
How can I predict feature vector from RNN?

### ISSUE THREAD

**Issue by MilkKnight** [fchollet/keras#2552]
Hi.
I want to predict the next time-step feature vector from a sequence of feature vectors.
For example, I have a time sequence [X1,X2,...,Xn], and each of the sequence is a feature vector. And I want to predict next feature vector Xn+1.
The loss function is cosine similarity.
I have try to use
`model.add(Activation('linear'))
model.compile(loss='cosine_proximity',optimizer='rmsprop')`
However, it didn't work. How can I fix it?
Thanks.

**Comment by carlthome** [comment]
Look at the examples. [This](https://github.com/fchollet/keras/blob/master/examples/stateful_lstm.py) would be a good start.
